src/translations/nl.json

```
@@ -44,6 +44,7 @@
                 "secondary_info": "Secundaire informatie",
                 "content_info": "Inhoud",
                 "use_entity_picture": "Gebruik entiteit afbeelding"
+                "collapsible_controls": "Bedieningselementen verbergen wanneer uitgeschakeld"
```

```suggestion
                "use_entity_picture": "Gebruik entiteit afbeelding",
                "collapsible_controls": "Bedieningselementen verbergen wanneer uitgeschakeld"
```
The build don't pass because of missing coma.